packages/spear-cli/package.json

```diff
@@ -40,7 +40,7 @@
     "url": "https://github.com/unimal-jp/spear/issues"
   },
   "dependencies": {
-    "@spearly/cms-js-core": "1.0.17",
+    "@spearly/cms-js-core": "1.0.19",
```

Comment:
The version already distributed was `1.0.18`.

https://www.npmjs.com/package/@spearly/cms-js-core?activeTab=versions